persistence/dynamodb/index.md

```diff
@@ -55,6 +55,22 @@ and registered on the container
 
 snippet: DynamoDBCustomClientProviderRegistration
 
+## Permissions
+
+Below is the list of minimum required [IAM policies for operating the DynamoDB persistence](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/security_iam_service-with-iam.html) with installers enabled: 
+
+  - `dynamodb:CreateTable`,
+  - `dynamodb:DescribeTable`,
+  - `dynamodb:DescribeTimeToLive`,
+  - `dynamodb:UpdateTimeToLive`,
+  - `dynamodb:Query`,
+  - `dynamodb:GetItem`,
+  - `dynamodb:BatchWriteItem`,
+  - `dynamodb:PutItem`,
+  - `dynamodb:DeleteItem`
+
+If installers are disabled, or if `DisableTableCreation` is called, the `dynamodb:CreateTable`, `dynamodb:DescribeTable` and `dynamodb:UpdateTimeToLive` policies are not required. 
```

Comment:
Since installers are not enabled by default, here's a suggestion to split the policies into two groups.

```suggestion
Below is the list of minimum required [IAM policies for operating the DynamoDB persistence](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/security_iam_service-with-iam.html) to use DynamoDB persistence:

  - `dynamodb:Query`,
  - `dynamodb:GetItem`,
  - `dynamodb:BatchWriteItem`,
  - `dynamodb:PutItem`,
  - `dynamodb:DeleteItem`
  
When installers are enabled (and table creation is not explicitly disabled via `DisableTableCreation`), the following permissions are required as well:

  - `dynamodb:CreateTable`,
  - `dynamodb:DescribeTable`,
  - `dynamodb:DescribeTimeToLive`,
  - `dynamodb:UpdateTimeToLive`,
```

I'm fine either way, feel free to chose what you feel works better. It's just a minor suggestion